KubeArmor/core/dockerHandler.go

```diff
@@ -237,6 +237,18 @@ func (dm *KubeArmorDaemon) GetAlreadyDeployedDockerContainers() {
 					continue
 				}
 
+				// check for unorchestrated docker containers
+				if !dm.K8sEnabled {
+					dm.ContainersLock.Lock()
+					container.ProcessVisibilityEnabled = true
```

Comment:
do we simply enable all visibility flags rather than using given visibility flags?

Reply:
Can you please mention what all other flags need to be enabled?

Reply:
Lets set the visibility based on the env var like we do in the case of k8s.